public/javascripts/Admin/src/Admin.js

```diff
@@ -434,8 +434,17 @@ function Admin(_, $) {
                 vega.embed("#severity-histograms", chart, opt, function(error, results) {});
                 vega.embed("#severity-histograms2", chart2, opt, function(error, results) {});
             });
-
             $.getJSON('/adminapi/neighborhoodCompletionRate', function (data) {
+                // Determine height of the chart based on the number of neighborhoods.
+                var chartHeight = 150 + (data.length * 30);
+
+                // Add css in order to allow for overflow to accomodate large number of neighborhoods.
+                var chartViewAdjust = $('#neighborhood-completion-rate');
+                chartViewAdjust.css({
```

Comment:
Instead of adding CSS in the JavaScript, you should just include it in our CSS files! I believe that `admin.css` is the one you'd use here.